e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/validation/company-details-company-website-validation.spec.js

```diff
@@ -37,6 +37,7 @@ describe("Insurance - Your business - Company details page - As an Exporter I wa
       referenceNumber = id;
 
       url = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.EXPORTER_BUSINESS.COMPANY_DETAILS}`;
+      natureOfBusinessUrl = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.EXPORTER_BUSINESS.NATURE_OF_BUSINESS}`;
```

Comment:
Not sure if we should do in this PR or afterwards, but it wasn't clear to me what/how this URL is used in these E2E tests, I can see now but I think it'd be clearer if we wrap these instances:

```js
submitButton().click();
cy.url().should('eq', natureOfBusinessUrl);
```

with:

```js
it(`should redirect to ${NATURE_OF_BUSINESS}`, () => { ... });
```